src/Reflex/Dom/Pandoc/Document.hs

```diff
@@ -244,3 +244,31 @@ renderInline cfg = \case
     inQuotes w = \case
       SingleQuote -> text "‘" >> w <* text "’"
       DoubleQuote -> text "“" >> w <* text "”"
+
+    -- Img alt text is represented as [Inline]
+    imageAttrs attr xs (iUrl, iTitle) =
+      sansEmptyAttrs $ renderAttr attr <> ("src" =: iUrl <> "title" =: iTitle <> "alt" =: inlinesToAlt xs)
+
+    inlinesToAlt = T.concat . map inlineToText
+
+    inlineToText = \case
+      Str t -> t
+      Emph xs -> inlinesToAlt xs
+      Underline xs -> inlinesToAlt xs
+      Strong xs -> inlinesToAlt xs
+      Strikeout xs -> inlinesToAlt xs
+      Subscript xs -> inlinesToAlt xs
+      SmallCaps xs -> inlinesToAlt xs
+      Quoted _ xs -> inlinesToAlt xs
+      Cite _ xs -> inlinesToAlt xs
+      Code _ t -> t
+      Space -> " "
+      SoftBreak -> " "
+      LineBreak -> " "
+      Math _ t -> t
+      RawInline _ t -> t
+      Link _ xs _ -> inlinesToAlt xs
+      Image _ xs _ -> inlinesToAlt xs
+      Note _ -> ""
+      Span _ xs -> inlinesToAlt xs
+      _ -> ""
```

Comment:
This looks like to be the same as [plainify](https://github.com/srid/neuron/blob/2956279f445b7bb462303f4c48ac74aba3d69b95/neuron/src/lib/Text/Pandoc/Util.hs#L73-L85) in neuron. I'll change it accordingly.